### PR TITLE
Refactor run script changes

### PR DESCRIPTION
**Pull request by MAJ0RRR**
I've added change from #43. It can be closed after this PR gets merged.

Now debug scripts can be launched in 3 modes:

1) Local

- ex. `./run_scripts/run-streams-collatz-debug L 7 1 30` - run locally with **7 slave proccesses** and **1 stream** for GPU with **30 as powercap** value

2) Remote

- ex `./run_scripts/run-streams-collatz-debug R 5 1 30` - run remotly on KASK server with **5 slave processes** (each on different machine, 5 first from the host file actually so des01 - des05) with **1 stream** for GPU and **30 as powercap** value

3) Choose

- ex `./run_scripts/run-streams-collatz-debug C 2 4 6 8 10 A 1 30` - run remotly on KASK server with **5 slave processes** (each on different machine: des02, des04, des06, des08, des10) with **1 stream** for GPU and **30 as powercap** value

  Basically mode followed by numbers of machines (des01 is 1, des02 is 2 and so on) followed by 'A' (now arguments for the app itself) so again streams count for GPU and the powercap value

### CODE REVIEW

**Comment by MAJ0RRR** [comment]
todo: unify into 1 file

**Comment by psienski** [comment]
I believe it would be a good idea to also add 'H' argument for syntax help. It might just print the same information as in PR comment.
